x/ccv/parent/keeper/keeper.go

```diff
@@ -476,14 +474,9 @@ func (h StakingHooks) UnbondingDelegationEntryCreated(ctx sdk.Context, delegator
 
 	// Set UBDE
 	h.k.SetUnbondingDelegationEntry(ctx, ubde)
```

Comment:
Replace `UnbondingDelegationEntry` with `UnbondingOperation` throughout the source code. Also the `UBDEIndex`, e.g., in the spec it's `vscToUnbondingOps`.